project/Gulp.scala

```diff
@@ -0,0 +1,61 @@
+import play.sbt.PlayRunHook
```

Comment:
What does this do? How should it be used - does it kick in automatically when you run `sbt test` etc or do you have to run `sbt gulp` explicitly. Does anything need to be installed?

Worth updating the README

We should also decide whether we want to serve assets from a dedicated frontend.

I've just tried `sbt test` and it seems this does kick in automatically - and fails on my machine even though I have node etc installed (wrong version?). This might be a reason to put it in its own repository (so that only devs working on the frontend assets need to have JavaScript tools installed).